.github/workflows/linux.yml

```diff
@@ -62,6 +62,42 @@ jobs:
           path:
             opensc*.tar.gz
 
+  build-arm64:
+    name: build-${{ matrix.dist }}-${{ matrix.arch }}
+    strategy:
+      matrix:
+        dist: ['22.04', '24.04']
+        arch: ['arm64']
+    runs-on: ubuntu-${{ matrix.dist }}${{ matrix.arch == 'arm64' && '-arm' || '' }}
```

Comment:
nit: We have the arch = arm64 here in all the cases so the condition is not needed, unless you would like to use this job to run some more x86_64 jobs and simplify this workflow file.